tests/examples_methods_syntax/multiline_tooltip.py

```diff
@@ -41,15 +41,16 @@
 ).add_params(
     nearest
 )
+when_near = alt.when(nearest)
 
 # Draw points on the line, and highlight based on selection
 points = line.mark_point().encode(
-    opacity=alt.condition(nearest, alt.value(1), alt.value(0))
+    opacity=when_near.then(alt.value(1)).otherwise(alt.value(0))
 )
 
 # Draw text labels near the points, and highlight based on selection
 text = line.mark_text(align="left", dx=5, dy=-5).encode(
-    text=alt.condition(nearest, "y:Q", alt.value(" "))
+    text=when_near.then("y:Q").otherwise(alt.value(" "))
```

Comment:
@joelostblom I decided this would be best left in the gallery in [docs: Add new subsection content](https://github.com/vega/altair/pull/3544/commits/50a520e9314108d572ef8bd6d48f0135151d2c6c)

All of the cases I've reused a (`When|Then`) have been for quite long/detailed examples - which seem excessive to include for just this small bit of functionality